roles/load-ycsb/tasks/main.yml

```diff
@@ -46,7 +46,7 @@
                 value: "{{uuid}}"
```

Comment:
you'll also need to export the es_index like you did for ycsb-bench role and before you do that you'll need to add it to defaults/main.yml similar to ycsb-bench.. 

```
              - name: es_index
                value: "{{ es_index }}"
```